lms/djangoapps/courseware/toggles.py

```diff
@@ -190,3 +190,18 @@ def streak_celebration_is_active(course_key):
         courseware_mfe_progress_milestones_are_active(course_key) and
         COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES_STREAK_CELEBRATION.is_enabled(course_key)
     )
+
+
+# .. toggle_name: COURSES_INVITE_ONLY
+# .. toggle_implementation: SettingToggle
+# .. toggle_type: feature_flag
+# .. toggle_default: False
+# .. toggle_description: Set this to manage the default value for INVITE_ONLY across all courses in a given deployment
```

Comment:
I'm not sure I 100% follow this description, and I'm not sure why. Maybe `Setting this sets the default value of INVITE_ONLY across all courses in a given deployment`